com.chartboost.helium.demo/ProjectSettings/AndroidResolverDependencies.xml

```diff
@@ -56,7 +56,7 @@
   </packages>
   <files />
   <settings>
-    <setting name="androidAbis" value="arm64-v8a,armeabi-v7a" />
+    <setting name="androidAbis" value="armeabi-v7a" />
```

Comment:
What is the `androidAbis` setting?

Reply:
Ah this is a mistake, since I modified some settings to only enable arm7 for faster builds, I will put it back.

Reply:
This just refers to the architectures the libs are supposed to resolve.